scalatags/shared/src/main/scala/scalatags/text/Pretty.scala

```diff
@@ -0,0 +1,56 @@
+package scalatags
+package text
+
+object Utils {
+
+  implicit class Pretty(underlying: Frag) {
```

Comment:
Rather than making this an implicit, could we make it an object inside `scalatags.Text.TypedTag`/`scalatags.Text.Frag`? I don't see any reason to have it in a separate import since it's already namespaced via needing `.pretty`